test/apple/fastfile.test.ts

```diff
@@ -46,4 +44,490 @@ describe('fastlane', () => {
       });
     });
   });
+
+  describe('#findIOSPlatform', () => {
+    describe('platform block detection', () => {
+      const variations: {
+        name: string;
+        content: string;
+        expected: { index: number; length: number };
+      }[] = [
+        {
+          name: 'no platform',
+          content: 'lane :test do\nend',
+          expected: { index: 0, length: 17 },
+        },
+        {
+          name: 'platform is ios',
+          content: 'platform: ios\nlane :test do\nend',
+          expected: { index: 0, length: 31 },
+        },
+        {
+          name: 'platform is ios and other platform',
+          content: 'platform: ios\nend\nplatform: android\nlane :test do\nend',
```

Comment:
`m`: I think it would be great to add test cases when the platform uses tabs and spaces before it. So basically a test for something like
```
content: '   platform: ios\nend\nplatform: android\nlane :test do\nend',
```

Then we know the regex also works when somebody has their code formatted with indentation.

Reply:
Do we support the case when the indentation is formatted?
Do we in general have any specs of cases we support?